consensus/enclave/impl/src/lib.rs

```diff
@@ -280,16 +280,10 @@ impl ConsensusEnclave for SgxConsensusEnclave {
         peer_id: &ResponderId,
         msg: PeerAuthResponse,
     ) -> Result<(PeerSession, VerificationReport)> {
-        // Inject the minimum fee (if necessary) before passing off to the AKE
-        let minimum_fee = self.minimum_fee.load(Ordering::SeqCst);
-        let session_and_report = if minimum_fee != MINIMUM_FEE {
-            let peer_id_str = format!("{}-{}", peer_id, minimum_fee);
-            self.ake.peer_connect(&ResponderId(peer_id_str), msg)?
-        } else {
-            self.ake.peer_connect(peer_id, msg)?
-        };
+        // Inject the if fee map hash passing off to the AKE
+        let peer_id = ResponderId(format!("{}-{}", peer_id, self.fee_map.get_digest_str()));
```

Comment:
Dunno if intended, but this does make the fee config mandatory.

Reply:
My thinking around it is that having less branches around whether a fee map has been provided or not results in simpler code, and the performance implication of that are negligible. Open to hearing otherwise.

Reply:
I thought about this a bit more, I think yours is superior.

Previously if the one fee was set to MINIMUM, then it would skip putting it on the responder ID. It didn't matter because all the enclaves were guaranteed to be the same version due to how n2n works anyways, but this is strictly better.